client/my-sites/email/inbox/new-mailbox-upsell/index.tsx

```diff
@@ -43,9 +47,16 @@ const NewMailboxUpsell = ( { domains }: { domains: ResponseDomain[] } ) => {
 	return (
 		<div className="new-mailbox-upsell__container">
 			<div className="new-mailbox-upsell">
+				{ isTrialNow ? <Ribbon color="green">FREE</Ribbon> : null }
```

Comment:
We can probably do the shorthand `isTrialNow ?? <Ribbon color="green">FREE</Ribbon>` here :)

Reply:
Good point, thank you :)
Only instead of nullish coalescing it's necessary to use logical &&, since the first one renders element only in case of null/undefined.

[Article with recommendation from React](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)

Reply:
I think we usually follow this approach for conditional blocks:

```suggestion
				{ isTrialNow && <Ribbon color="green">FREE</Ribbon> }
```

Reply:
@stephanethomas, thank you.
As I already mentioned - it's recommended way by React.
[Link for documentation](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator).

Reply:
I was discussing this with @nightnei in our onboarding session. II agree that is better to use a logical && coalesce. Our code is plenty of `condition && <Element/>` and probably the best thing to do here is basically to change it like this:
```suggestion
				{ isTrialNow && <Ribbon color="green">FREE</Ribbon> }
```